src/ontology/uberon-edit.obo

```diff
@@ -117782,12 +117783,14 @@ property_value: external_ontology_notes "in FMA the Adbuctor muscle is a subclas
 id: UBERON:0006846
 name: surface groove
 def: "A furrow or an incomplete tube." [AEO:0000161, AEO:JB, https://github.com/obophenotype/human-developmental-anatomy-ontology/issues/4]
+comment: Surface groove are modelled as immaterial - see https://github.com/obophenotype/uberon/issues/2667 for discussion
 subset: grouping_class
 synonym: "groove" RELATED [AEO:0000161]
 xref: AEO:0000161
 xref: EHDAA2_RETIRED:0003161
-intersection_of: UBERON:0003102 ! surface structure
-intersection_of: has_quality PATO:0002255 ! grooved
+is_a: UBERON:0000464 ! anatomical space
```

Comment:
I think grooves are adjacent to space (in line with the definition of a 'surface structure'), but can appreciate other perspectives. I'm inclined to classify groove as a 'surface structure'.

Reply:
ahahahaha its exactly what I'm trying to get rid of, surface structures are material. I think if we model sulcus, incisure, and fissures as immaterial, we should model grooves as immaterial too. I think we can say that it is adjacent surface structure instead?

Reply:
>  I think if we model sulcus, incisure, and fissures as immaterial, we should model grooves as immaterial too.

Yes, my perspective is that these are properties describing the orientation of materials in space and do not have mass.